### PR TITLE
Update to 40.0 and add x-checker-data annotations

### PR DESCRIPTION
**Pull request by sophie-h**
Supersedes #18

### CODE REVIEW

File: org.gnome.Contacts.yml [hunk 1]
```diff
@@ -123,8 +123,8 @@ modules:
       - /share/GConf
     sources:
       - type: archive
-        url: https://download.gnome.org/sources/folks/0.14/folks-0.14.0.tar.xz
-        sha256: d6e778da637d3b33fd9a6b45115e8157d8868211dc08b943d1075b5eb7282bb9
+        url: https://download-fallback.gnome.org/sources/folks/0.15/folks-0.15.2.tar.xz
```
Comment (nielsdg) [inline org.gnome.Contacts.yml]: any reason we're using download-fallback here instead of download.gnome.org? :)

File: org.gnome.Contacts.yml [hunk 2]
```diff
@@ -166,5 +166,5 @@ modules:
       - --buildtype=debugoptimized
     sources:
       - type: archive
-        url: https://download.gnome.org/sources/gnome-contacts/3.38/gnome-contacts-3.38.1.tar.xz
-        sha256: c96dce19de25d99a219bd52dfff9746226663d0d99d4506d570c2ad979035843
+        url: https://download-fallback.gnome.org/sources/gnome-contacts/40/gnome-contacts-40.0.tar.xz
```
Comment (nielsdg) [inline org.gnome.Contacts.yml]: Same question here (why the -fallback)

File: org.gnome.Contacts.yml (hunk 1 above)
Comment (sophie-h) [reply org.gnome.Contacts.yml]: I don't think there is a reason. There are no more fallback links in the final diff.

**Comment by nielsdg** [comment]
Thanks a lot! :-)

**Pull request merged**